pkg/apis/oauth/handler.go

```diff
@@ -128,6 +128,7 @@ func (h *handler) passwordGrant(username string, password string, req *restful.R
 	authenticated, provider, err := h.passwordAuthenticator.Authenticate(username, password)
 	if err != nil {
 		formatErr := fmt.Errorf("incorrect username or password")
+		rateLimitErr := fmt.Errorf("the account has been locked because of excessive password errors. please try again after 30 minutes or contact the administrator to reset your password")
```

Comment:
"or contact the administrator to reset your password" remove

"30 minutes" Replace the “AuthenticateRateLimiterDuration” parameter in the configuration file with 

fmt.Sprintf("the account has been locked because of excessive password errors. please try again after %s minutes", AuthenticateRateLimiterDuration)